scene/audio/audio_stream_player.cpp

```diff
@@ -392,6 +412,8 @@ void AudioStreamPlayer::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::OBJECT, "stream", PROPERTY_HINT_RESOURCE_TYPE, "AudioStream"), "set_stream", "get_stream");
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "volume_db", PROPERTY_HINT_RANGE, "-80,24"), "set_volume_db", "get_volume_db");
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "pitch_scale", PROPERTY_HINT_RANGE, "0.01,4,0.01,or_greater"), "set_pitch_scale", "get_pitch_scale");
+	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "volume_scale_l", PROPERTY_HINT_RANGE, "0.0,11,0.0,or_greater"), "set_volume_scale_l", "get_volume_scale_l");
+	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "volume_scale_r", PROPERTY_HINT_RANGE, "0.0,11,0.0,or_greater"), "set_volume_scale_r", "get_volume_scale_r");
```

Comment:
Does the maximum allowed value of `11` have any specific meaning? I'm not sure if the inspector should allow setting values above 1.0. Also, the `or_greater` part may not be needed here.

Reply:
It hasn't got any specific meaning, the point was to allow arbitrarily high volume scales. What's the correct mechanism for this?

Reply:
Since `volume_db` already allows for adjusting volume above the default volume, I wouldn't expose another way to do this in the inspector. It can end up being confusing if you use the individual volume scales for anything else than manual panning.

Reply:
I've changed it to "0.0,1.0" in latest commit.